src/components/sections/Hero.section.tsx

```diff
@@ -34,18 +34,16 @@ const HeroSection: React.FC = () => {
     return (
         <div className="bg-midnight text-white py-16">
           <div className="text-center">
-            <h1 className="text-6xl font-bold mb-4">
-             <span className="text-mint">&lt;HawkHacks 2024/&gt;</span>           
-            </h1>
+            <h1 className="font-bold text-mint">&lt;HawkHacks 2024/&gt;</h1>           
             <p className="pb-2 text-lg">Wilfrid Laurier’s first large-scale global hackathon!</p>
             <p className="pb-3 text-xs font-thin [word-spacing:5px]">Waterloo, ON | April 6th - 8th | In-Person</p>
-            <p className="mx-5 space-x-8 text-2sm font-bold [word-spacing:45px]">{countdown}</p>
-            <p className="pb-3 text-sm font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
-            <button className="bg-mint text-white px-4 py-1 rounded">
+            <p className="mx-5 space-x-8 text-base font-bold [word-spacing:45px]">{countdown}</p>
+            <p className="pb-3 text-sm mb-2 font-thin [word-spacing:35px] mr-[-0.5rem]">Days Hrs Mins Secs</p>
+            <button className="bg-mint text-white mb-3 px-4 py-2 rounded font-bold [letter-spacing:2px]">
               REGISTER
             </button>
-            <img  className="mt-[-4rem]" src={HeroIllustration} alt="Hero Illustration" />
-            <div className="flex items-center justify-center space-x-4 mt-[-8.5rem]">
+            <img  className="mt-[-0.5rem]" src={HeroIllustration} alt="Hero Illustration" />
```

Comment:
```suggestion
            <img  className="-mt-2" src={HeroIllustration} alt="Hero Illustration" />
```

Reply:
Tailwind has classes for negative values. Little tip here: all measures in tailwind start with 0.25rem. So mt-1 = 0.25rem and mt-2 = 0.5rem.

Reply:
You can also use half points such as `mt-0.5` to get 0.125rem